File: MPI/mpicheck/tests/functions/1.0/MPI_Cart_map.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    int var_1;
    const int var_2[2];
    const int var_3[2];
    int *var_4;
    int ret;
    /* calls */
    ret = MPI_Cart_map(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_Cart_map(var_0, var_1, var_2, var_3, var_4);
    return 0;
}

File: MPI/mpicheck/tests/functions/1.0/MPI_Cart_map.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2(10)
       LOGICAL var_3(10)
       INTEGER var_4
       INTEGER var_5
        call mpi_cart_map(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_cart_map(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    